src/native-common/Text.tsx

```diff
@@ -47,6 +60,17 @@ export class Text extends React.Component<Types.TextProps, {}> {
         );
     }
 
+    private _onNativeTextRef = (ref: RN.Text|null) => {
+        this._nativeText = ref;
+    }
+
+    getChildContext() {
+        // Let descendant Types components know that their nearest Types ancestor is an Types.Text.
```

Comment:
>Types [](start = 26, length = 5)

I think all occurrences of `Types` here should be `RX`

Reply:
ok

---
In reply to: [159348943](https://github.com/Microsoft/reactxp/pull/404#discussion_r159348943) [](ancestors = 159348943)